sspl_test/alerts/node/test_node_bmc_interface.py

```diff
@@ -59,13 +51,10 @@ def test_bmc_interface(args):
             time.sleep(0.1)
             print(exception)
 
-    #restore bmc config and activate ipmisimtool
-    simulate_bmc_interface_alert.restore_config()
 
     assert(bmc_interface_message is not None)
     assert(bmc_interface_message.get("alert_type") is not None)
-    alert_type = bmc_interface_message.get("alert_type")
-    assert(alert_type=="fault")
+    assert(bmc_interface_message.get("alert_type") == alert_type)
```

Comment:
All tests can be upgraded to read site_id, rack_id,node_id, cluster_id from sspl config & assert with alert data. Start with this test itself.